ground_z: -0.04
#ground_z: -0.07
X_tcp_cage:
  x: 0.000548455
  y: 0.00354947
  z: -0.194698
flu_T_tcp:
  x: -0.07
  y: 0
  z: -0.114698
insert_below_tip: 0.1
rnw:
  hover_above_tip: 0.03
  insertion_depth: 0.12 # 0.11
  topple_init: 0.05
  desired_nutation: 30
  desired_grip_depth: 0.05
  tau: 20
  max_vel: 0.5
  max_acc: 0.3
  rocking_max_vel: 1
  rocking_max_acc: 0.5
  adjust_nutation_threshold: 5
  adjust_grip_depth_threshold: 0.05
  ang_vel_threshold: 0.3
  min_nutation_deg: 5
  yaw_gain: 0.2
  desired_spin_deg: 30
  spin_Ki: 0.1
  spin_Kp: 0.1
  lap_ang_vel_deg: 0
  lap_start: 2
  enable_steering: true
  enable_energy_feedback: true
  EKp: 0.2
  EKi: 0
  EKd: 0
  tau_ff: 0.1
topple:
  forward: 0.4
  downward: 0.05
zigzag:
  step_forward: 0.1
  step_sideways: 0.1
  cycles: 5
  max_vel: 1
  max_acc: 0.5
cone:
  radius: 0.15
  base_center:
    x: -0.00950682
    y: -0.00453934
    z: -0.053012
  tip:
    x: 0.140446
    y: -0.00314827
    z: 0.949205
  CoM_x: 0.08
  CoM_z: 0.05
  mass: 0.5